doc/source/guide/installation/linux.rst

```diff
@@ -5,11 +5,16 @@ Linux
 Overview
 --------
 
+.. warning::
```

Comment:
have we put one of these in the windows section?

I would also maybe reword this a little bit, especially for Linux where there are other good package management options.

Maybe something like:

```
We highly recommend that linux users install the Anaconda python distribution which is described in the instructions here :ref:`anaconda_install`.
```